server/autotest_server/testers/__init__.py

```diff
@@ -1,13 +1,16 @@
 import os
 
-_TESTERS = ("custom", "haskell", "java", "jupyter", "py", "pyta", "r", "racket")
+from .models import TesterType
 
 
-def install(testers=_TESTERS):
+def install(testers_to_install: list[TesterType]) -> tuple[list[TesterType], dict[TesterType, dict]]:
+    """Tries to install testers. Returns successfully installed testers and their settings."""
```

Comment:
There should be some clarification here that these are settings that are obtained at runtime.